google-api-client/src/main/java/com/google/api/client/googleapis/media/MediaHttpUploader.java

```diff
@@ -351,6 +360,7 @@ private HttpResponse directUpload(GenericUrl initiationRequestUrl) throws IOExce
       initiationRequestUrl.put("uploadType", "multipart");
     } else {
       initiationRequestUrl.put("uploadType", "media");
+      initiationRequestUrl.put("contentOffset", String.valueOf(contentOffset));
```

Comment:
Should this be set to `-1` if `contentOffset` is not set by the user? Seems like this is new, unexpected behavior unless there's a specification for uploads saying to contentOffset of -1 is normal.